src/pages/BeerDetails.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { Link, NavLink } from "react-router-dom";
+
+function BeersList(props) {
+  const { beers } = props;
+  return (
+    <div>
+      {beers.map((beer) => {
+        return (
+          <div>
+            <Link to={`/${beer._id}`}>
+              <p>{beer.name}</p>
+            </Link>
+          </div>
+        );
+      })}
+    </div>
+  );
+}
+
+export default BeersList;
```

Comment:
And if you change the name of the function you have to change it here on the export as well